crates/generate-raw/src/lib.rs

```diff
@@ -199,6 +242,18 @@ impl Render for BuiltinType {
 
 impl Render for Module {
     fn render(&self, src: &mut String) {
+        let rust_name = self.name.as_str().to_snake_case();
+        // wrapper functions
+        for f in self.funcs() {
+            render_highlevel(&f, &rust_name, src);
+            src.push_str("\n\n");
+        }
+
+        // raw module
+        src.push_str("#[allow(non_camel_case_types)]\n");
```

Comment:
Do we still need `non_camel_case_types`?

Reply:
We do not!